spec/dummy/config/initializers/wicked_pdf.rb

```diff
@@ -0,0 +1,22 @@
+# WickedPDF Global Configuration
```

Comment:
I think this may need to be moved to config/initializers so that it is active in the other projects

Reply:
its moved